landlab/components/soil_grading/soil_grading.py

```diff
@@ -386,11 +386,11 @@ def create_dist(
         else:
 
             total_bedrock_weight = 10000  # There is no meaning for this number
-            if median_size == None:
+            if median_size is None:
```

Comment:
Suggest adding a docstring to this function to explain briefly what it's for.

Reply:
Also, some more explanation to go with the comment, "There is no meaning to this number" would be good. 😄

Reply:
Will do